pkg/plugins/platform/local.go

```diff
@@ -960,10 +960,10 @@ func GetActiveAppRoot(siteName string) (string, error) {
 			return "", fmt.Errorf("could not determine the location of %s from container: %s", siteName, dockerutil.ContainerName(webContainer))
 		}
 	}
-
 	appRoot, err := CheckForConf(siteDir)
 	if err != nil {
-		return "", fmt.Errorf("unable to determine the application for this command. Have you run 'ddev config'? Error: %s", err)
+		// In the case of a missing .ddev/config.yml just return the site directory.
+		return siteDir, nil
```

Comment:
Yes because it will still get the app root from the running web container and return it by itself if it can't find a config.

Reply:
This is the message shown with this PR in the situation @rfay describes (ddev start on dir that config has not been run for): "Failed to start: Could not find a site in /Users/tannerjfco/sites/test. Please specify a site name or change directories: no .ddev/config.yaml file was found in this directory or any parent" This may be ok enough, but we no longer suggest to run config. Maybe change to "Could not find a configured site?"